PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -9,22 +11,34 @@ public static void Main(string[] args)
         {
             string filePath = GetFilePath();
             Question[] questions = LoadQuestions(filePath);
-
+            
+            Question[] questionsRnd = RandomizeQuestions(questions);
```

Comment:
```suggestion
            Question[] questionsRandom = RandomizeQuestions(questions);
```
It's best to avoid abbreviations in your variable names, as it can make your code harder to read.